draft-ietf-moq-transport.md

```diff
@@ -912,8 +912,7 @@ usually indicate that state can immediately be destroyed, see
 
 The Publisher can destroy subscription state as soon as it has received
 UNSUBSCRIBE. It MUST reset any open streams associated with the SUBSCRIBE.
-
-The publisher can immediately delete subscription state after sending
+The Publisher can also immediately delete subscription state after sending
 PUBLISH_DONE, but MUST NOT send it until it has closed all related streams.
```

Comment:
This makes sense. but not sure if this can be enforced though . Asking differently, if the publisher sends Done before closing all the streams, will it be an error or protocol violation ?